x-pack/plugins/security_solution/public/entity_analytics/components/risk_information/index.tsx

```diff
@@ -200,37 +227,68 @@ export const RiskInformationFlyout = ({ handleOnClose }: { handleOnClose: () =>
               />
             </h3>
           </EuiTitle>
-          <p>
+          <ol>
+            <li>
+              <FormattedMessage
+                id="xpack.securitySolution.riskInformation.riskCalculationStep1"
+                defaultMessage="Only scores entities (users and hosts) associated with a detection alert, based on any alerts that have not been closed."
+              />
+            </li>
+            <li>
+              <FormattedMessage
+                id="xpack.securitySolution.riskInformation.riskCalculationStep2"
+                defaultMessage="Generates an 'Alert' category score by aggregating alerts by entity idenfifier such that alerts with higher risk contribute more than alerts with lower risk scores."
```

Comment:
I took this verbatim from the ticket, but I think this reads a little better: 

```suggestion
                defaultMessage="Generates an 'Alert' category score by aggregating alerts by entity idenfifier such that alerts with higher risk scores contribute more than alerts with lower risk scores."
```

Reply:
👍 agree

Reply:
cc @SourinPaul 👀